assert/assert_strict_equals.ts

```diff
@@ -8,6 +8,12 @@ import { red } from "@std/fmt/colors";
  * Make an assertion that `actual` and `expected` are strictly equal. If
  * not then throw.
  *
+ * This function copies the behavior of Node's
+ * {@linkcode https://nodejs.org/api/assert.html#assertstrictequalactual-expected-message | assert.strictStrictEqual},
```

Comment:
We also consider the compatibility to jest, vitest, etc. I don't think we need to mention this information here